src/main/java/cristiano/Ronaldo.java

```diff
@@ -0,0 +1,194 @@
+        System.out.println("SIUUUUUUU!");
+    }
+
+    public void reject(String words) {
+        switch (words) {
+        case "Format":
+            System.out.println("""
+                    Invalid format! Please enter either of these commands in this format without '<>':\s
+                    <bye>\s
+                    <list>\s
+                    <mark> <goal number>\s
+                    <unmark> <goal number>\s
+                    <event> <description> /from <time> /to <time>\s
+                    <todo> <description>\s
+                    <deadline> <description> /by <time>\s
+                    """);
+            break;
+        case "Empty":
+            System.out.println("Woops, your input is empty, just like Spurs' trophy cabinet.\n");
+            break;
+        case "Range":
+            System.out.println("Goal number is out of range!\n");
+            break;
+        case "Mark/Unmark":
+            System.out.println("Invalid mark format! Please use: <mark>/<unmark> <goal number>\n");
+            break;
+        case "Marked":
+            System.out.println("Goal is already marked!\n" + " ");
+            break;
+        case "Unmarked":
+            System.out.println("Goal is already unmarked!\n" + " ");
+            break;
+        case "Event":
+            System.out.println("Invalid event format! Please use: <event> <description> /from <time> /to <time>\n");
+            break;
+        case "Deadline":
+            System.out.println("Invalid deadline format! Please use: <deadline> <description> /by <time>\n");
+            break;
+        case "Todo":
+            System.out.println("Invalid todo format! Please use: <todo> <description>\n");
+            break;
+        default:
+            break;
+        }
+    }
+
+    /**
+     * This method handles the marking and unmarking of a task/goal.
+     * The valid format to successfully mark or unmark a goal is: [command] [integer]
+     * Where command = mark/unmark, followed by a single space, followed by a valid integer.
+     * If an invalid format is used, a NumberFormatException message will be prompted.
+     * If the integer provided is out of bounds, an IndexOutOfBoundsException message will be prompted.
+     * If the user tries to mark or unmark a goal that has already been marked or unmarked,
+     * this method from the chatbot will remind the user its goal completion status.
+     * Goals completed will be marked as 'X'.
+     *
+     * @param input The goal index to be marked or unmarked;
+     */
+    public void handleGoal(String[] input) {
+        try {
+            if (input.length <= 1) {
+                reject("Mark/Unmark");
+                return;
+            }
+            int taskNumber = Integer.parseInt(input[1]) - 1;
+            Goal goal = goals.get(taskNumber);
+            if (input[0].equals("mark")) {
+                goal.markAsDone(this);
+            } else if (input[0].equals("unmark")) {
```

Comment:
Try to avoid magic literals. You can consider using named constants, especially when a particular number or string is used multiple times in the code.